index.js

```diff
@@ -15,6 +15,11 @@ var PluginError = gutil.PluginError;
 var PLUGIN_NAME = 'gulp-istanbul';
 var COVERAGE_VARIABLE = '$$cov_' + new Date().getTime() + '$$';
 
+function getNormalizedPath(filepath) {
```

Comment:
I plugged in the `normalize-path` module, but as I feared it didn't fully function. It did better than without it, but for some reason the number of statements passing coverage dropped by 50%.

Also, in the generated report the paths included the entire file path including hard drive letter rather than a relative path based on the root project directory. Versus using the original platform path separator which makes it work the same on Windows as it does on Mac/linux...